site/src/sphinx/server-decorator.rst

```diff
@@ -135,6 +135,70 @@ a certain type from a server:
         }
     }
 
+.. _server-decorator-service-with-path-mappings:
+
+Decorating ``ServiceWithPathMappings``
+--------------------------------------
+
+:api:`ServiceWithPathMappings` is a special variant of :api:`Service` which allows a user to register multiple
+routes for a single service. It has a method called ``pathMappings()`` which returns a ``Set`` of
+:apiplural:`PathMapping` so that you do not have to specify path mappings when registering your service:
+
+.. code-block:: java
+
+    public class MyServiceWithPathMappings implements ServiceWithPathMappings<HttpRequest, HttpResponse> {
```

Comment:
Seems like we missed some import statements throughout this document.
Such as for `ServiceWithPathMappings`, `ServiceRequestContext` and `PathMapping`.

Reply:
Thanks for the heads up. Added.